examples/disaggregated/README.md

```diff
@@ -200,6 +200,63 @@ srun -A <account> -p <partition> -t <time> \
 
 Additionally, we offer a fully executable script—please refer to [Disaggregated SLURM Scripts](./slurm/simple_example/).
 
+## Mixed Precision Context and Generation
+
+In disaggregated serving, the context (prefill) workers and generation (decode) workers have different performance characteristics: prefill workers are more compute-bound while decode workers are more memory-bound. Therefore, it may be beneficial to run prefill workers in higher precision. Running these workers with different precisions also enables the ability to interpolate between performance/compute trade-offs of different quantization levels.
```

Comment:
No need to specify prefill/decode, the terms are not used in the rest of the document.

Reply:
Changed!